### DISCUSSION TITLE
How to add more shortcuts above windows power button in win10 style menu?

### DISCUSSION THREAD

**Discussion by LoserForever88** [valinet/ExplorerPatcher#1974] [General]
As titled, those shortcuts like Documents, Settings etc were available in win10. Is there a way to bring it back?  The empty space seems kind of a waste by not filling shortcuts there.

![image](https://github.com/valinet/ExplorerPatcher/assets/122156717/4d315b70-1b42-40c3-9e1e-0545569e9a98)

**Comment by pyrates999** [comment]
Here's how you do it in windows 10: https://www.howtogeek.com/694939/how-to-add-folder-shortcuts-to-the-start-menus-left-sidebar-on-windows-10/

My guess is you'll need to figure out the registry settings that apply these settings and hope they still work for windows 11.

**Comment by LoserForever88** [comment]
> Here's how you do it in windows 10: https://www.howtogeek.com/694939/how-to-add-folder-shortcuts-to-the-start-menus-left-sidebar-on-windows-10/
> 
> My guess is you'll need to figure out the registry settings that apply these settings and hope they still work for windows 11.

I tried it.  Yeah, apparently changing those settings alone doesn't work :(

**Reply by pyrates999** [reply]
Then either the functionality was removed, or just disabled.  If it was just disabled, then it might be able to be added back into EP.  If it was removed, then it cannot be added back into EP.

**Comment by milestprower92** [comment]
Open regedit and go to "HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\CloudStore\Store\Cache" And Delete The Default Account Key and Restart Explorer

**Reply by Jazzzed** [reply]
What exactly does this do? Happy to try, but I’m not deleting registry keys unless I know what the expected outcome is supposed to be.

**Comment by milestprower92** [comment]
> What exactly does this do? Happy to try, but I’m not deleting registry keys unless I know what the expected outcome is supposed to be.

@Jazzzed What This Does Is Reset The Windows 10's Start Menu To it's original Layout 
![image](https://github.com/valinet/ExplorerPatcher/assets/131640765/b388bc3d-2911-48f2-ac35-8f692544a220)

**Reply by milestprower92** [reply]
but Without The Bloatware Because in Windows 11 22H2 The Code For The Download Arrows Has Been Removed From The Windows 10 Styled Start Menu

**Reply by Jazzzed** [reply]
Does it fix the problem reported by the OP?

**Reply by milestprower92** [reply]
@Jazzzed yes

**Reply by milestprower92** [reply]
because the default layout on the windows 10 start menu has Power Settings Documents Pictures & The User Icon

**Reply by milestprower92** [reply]
![image](https://github.com/valinet/ExplorerPatcher/assets/131640765/edbfd89c-5715-4fe1-9783-ba62482473e4)

**Reply by genlosha** [reply]
Would it be possible to add the Downloads folder to this list using the registry?

**Reply by Blackdrazon** [reply]
Unfortunately, I don't know any way to get the Downloads folder into the menu yet, which is too bad because it's the one I want the most.

Also, I'm not a registry expert, but I might be able to improve the fix a little. Instead of deleting the entire Cache folder, look underneath "HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\CloudStore\Store\Cache\DefaultAccount". You'll see a bunch of keys that start with a $ and some numbers (I think they're tied to your user account so I'm not going to copy and paste mine). One of those keys ends with "windows.data.unifiedtile.startglobalproperties". I was able to delete that key alone to get the fix while keeping my other settings.
